docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -0,0 +1,112 @@
+.. _installing_distros:
+
+Installing Ansible on specific operating systems
+================================================
+
+Follow these instructions to install the Ansible community package on a variety of operating systems.
+
+.. contents::
+  :local:
+
+Installing Ansible on Fedora or CentOS 
+----------------------------------------------
+
+On Fedora:
+
+.. code-block:: bash
+
+    $ sudo dnf install ansible
+
+On CentOS:
+
+.. code-block:: bash
+
+    $ sudo yum install epel-release
+    $ sudo yum install ansible
+
+RPMs for currently supported versions of CentOS are also available from `EPEL <https://fedoraproject.org/wiki/EPEL>`_.
+
+Ansible can manage older CentOS operating systems that contain Python 2.6 or later.
+
+.. _from_apt:
+
+Installing Ansible on Ubuntu
+----------------------------
+
+Ubuntu builds are available `in a PPA here <https://launchpad.net/~ansible/+archive/ubuntu/ansible>`_.
+
+To configure the PPA on your system and install Ansible run these commands:
+
+.. code-block:: bash
+
+    $ sudo apt update
+    $ sudo apt install software-properties-common
+    $ sudo add-apt-repository --yes --update ppa:ansible/ansible
+    $ sudo apt install ansible
+
+.. note:: On older Ubuntu distributions, "software-properties-common" is called "python-software-properties". You may want to use ``apt-get`` rather than ``apt`` in older versions. Also, be aware that only newer distributions (that is, 18.04, 18.10, and later) have a ``-u`` or ``--update`` flag. Adjust your script as needed.
+
+You can also build Debian/Ubuntu packages from the source checkout.
+
+.. code-block:: bash
+
+    $ make deb
+
+Installing Ansible on Debian
+----------------------------
+
+Debian users can use the same source as the Ubuntu PPA (using the following table).
+
+.. list-table::
+  :header-rows: 1
+
+  * - Debian
+    -
+    - Ubuntu
+  * - Debian 11 (Bullseye)
```

Comment:
Based on the existing patterns, I would expect Debian 11 to match Ubuntu jammy (22.04) that should be released anytime soon. Is that correct @Ompragash ?